flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/DispatcherThreadFactory.java

```diff
@@ -29,21 +31,41 @@
 	private final ThreadGroup group;
 	
 	private final String threadName;
+
+	private final ClassLoader classLoader;
 	
 	/**
 	 * Creates a new thread factory.
-	 * 
+	 *
 	 * @param group The group that the threads will be associated with.
 	 * @param threadName The name for the threads.
 	 */
 	public DispatcherThreadFactory(ThreadGroup group, String threadName) {
+		this(group, threadName, null);
+	}
+
+	/**
+	 * Creates a new thread factory.
+	 *
+	 * @param group The group that the threads will be associated with.
+	 * @param threadName The name for the threads.
+	 * @param classLoader The {@link ClassLoader} to be set as context class loader.
+	 */
+	public DispatcherThreadFactory(
+		ThreadGroup group,
```

Comment:
This is a code style preference rather than an issue, but I would suggest to indent the arguments by a tab to separate them from the body of the method.

Reply:
Indented.